packages/nimble-components/src/table-column/base/index.ts

```diff
@@ -0,0 +1,31 @@
+import type { ElementStyles, ViewTemplate } from '@microsoft/fast-element';
+import { FoundationElement } from '@microsoft/fast-foundation';
+import type { TableCellState, TableRecord } from '../../table/types';
+
+/**
+ * The base class for table columns
+ */
+export abstract class TableColumn<
+    TCellData extends TableRecord = TableRecord,
+    TColumnConfig = void
+> extends FoundationElement {
+    // The template to use to render the cell content for the column
+    public abstract cellTemplate: ViewTemplate<
+    TableCellState<TCellData, TColumnConfig>
+    >;
+
+    // The style to apply to the cellTemplate
+    public cellStyles?: ElementStyles;
```

Comment:
Any reason this is the only member not marked `abstract`?

Reply:
Because I don't believe it's technically necessary to implement? We could still mark it as abstract and just force an implementation to provide an empty style if that's appropriate.